heketi-centos-ci-tests.sh

```diff
@@ -40,7 +40,8 @@ yum -y install \
 	python-virtualenv \
 	ansible \
 	libvirt \
-	libvirt-devel
+	libvirt-devel \
+        ruby ruby-devel
```

Comment:
Separate out ruby and ruby-devel to different lines

Reply:
done